.config/tsaoptions.json

```diff
@@ -0,0 +1,8 @@
+{
+  "instanceUrl": "https://msazure.visualstudio.com",
+  "projectName": "One",
+  "areaPath": "One\\MGMT\\Compute\\Powershell\\Powershell",
```

Comment:
shouldn't this have a specific areaPath for secret management?
(ex. text utility is "One\\MGMT\\Compute\\Powershell\\Powershell\\PowerShell Core\\TextUtility")

Reply:
Heh, if that's the case, that changed since I started setting these up.